langgraph_supervisor/supervisor.py

```diff
@@ -161,9 +210,11 @@ def create_supervisor(
         output_mode: Mode for adding managed agents' outputs to the message history in the multi-agent workflow.
             Can be one of:
             - `full_history`: add the entire agent message history
+            - `only_history`: add the message history WITHOUT the supervisor's AI message and handoff messages.
```

Comment:
i think it should say without ALL handoff messages, right?

Reply:
ha ya